lib/apiservers/portlayer/swagger.json

```diff
@@ -637,6 +637,67 @@
 					}
 				}
 			},
+			"post": {
+				"description": "Join an image to a container",
```

Comment:
nit: Can we describe this more... usefully? (If someone doesn't know what "ImageJoin" does, "Join an image" probably isn't going to clarify things.) Perhaps you could steal a different phrasing from you package documentation?